src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__minute-hand {
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%);
+    transform-origin: bottom;
+    z-index: -1;
+    animation: move-minute-hand 3600s steps(60) infinite;
+  }
+
+  &__second-hand {
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%);
+    transform-origin: bottom;
+    z-index: -2;
+    animation: move-second-hand 60s linear infinite;
```

Comment:
Similar to the previous comment, `z-index: -2;` on line 46 may cause the `.stopwatch__second-hand` to appear below other elements. Ensure that this stacking order is intentional, and if not, adjust the `z-index` value accordingly.